build-tools/packages/build-cli/package.json

```diff
@@ -26,23 +26,21 @@
   ],
   "scripts": {
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint",
-    "build:compile": "npm run tsc",
+    "build:compile": "rimraf dist && npm run tsc",
     "build:full": "npm run format && npm run build && npm run build:manifest && npm run build:readme",
-    "build:full:compile": "npm run build:compile",
+    "build:full:compile": "npm run build:full",
```

Comment:
Do we need this one for consistency with other packages/tools? Otherwise why not just call build:full directly?